openacademy/views/openacademy_course_view.xml

```diff
@@ -22,13 +23,14 @@
         </field>
     </record>
 
-    <record model="ir.ui.view" id="course_tree_view">
+    <record model="ir.ui.view" id="course_list_view">
         <field name="name">course.list</field>
         <field name="model">openacademy.course</field>
         <field name="arch" type="xml">
             <tree>
                 <field name="name" string="Name Tree jj"/>
```

Comment:
The string is not necessary. Since in the model this field has a string  [attribute](https://github.com/jjbenavaz/openacademy-repo/pull/2/files#diff-ccd022b7e30bade2805e4dcf79a98f796bc05a2a29f3e370cf92a8b0d03e1982R9)